lib/resque/data_store.rb

```diff
@@ -76,10 +76,10 @@ def identifier
       @redis.inspect
     end
 
-    # Force a reconnect to Redis without closing the connection in the parent
-    # process after a fork.
-    def reconnect
-      @redis._client.connect
+    # Close the connection. Must be done after fork to avoid sharing the connection
+    # with the parent.
+    def disconnect!
```

Comment:
Is there any way to retain the reconnect functionality? Changing the method name on a public method might necessitate more of a version bump than I was hoping for right now. Additionally, it's not clear if after calling `disconnect!` if we're going to reconnect at a later time.

Reply:
The Redis client always reconnected automatically.

If you want to force a reconnection, you can issue a command, e.g. `PING`.

Reply:
I added `reconnect` back as an alias, it's functionally equivalent unless somehow you 100% rely on the connection being create right there and not on use.